share/governance.md

```diff
@@ -0,0 +1,80 @@
+
+VoxPupuli Governance
+====================
+
+
+Roles and Responsibilities
+--------------------------
+
+In order to have a smoothly running project, formal roles with corresponding responsibilities are established. A member of the community may have multiple roles.
+
+
+Users
+-----
+
+
+Users are community members who have a need for the project. They are the most important members of the community: without them, the project would have no purpose. Anyone can be a user; there are no specific requirements. Users should be encouraged to participate in the life of the project and the community as much as possible. User contributions enable the project team to ensure that they are satisfying the needs of those users. Common user activities include (but are not limited to):
+
+* advocating for use of the project
+* informing developers of project strengths and weaknesses from a new user’s perspective
+* providing moral support (a ‘thank you’ goes a long way)
+* writing documentation and tutorials
+* filing bug reports and feature requests
+* participating on the discussion board
+
+
+Users who continue to engage with the project and its community will often find themselves becoming more and more involved. Such users may then go on to become contributors, as described below.
+
+*How to become one*: Use a VoxPupuli project
+
+
+Collaborators
+-------------
+
+
+Collaborators are community members who submit patches to the project. These patches may be a one-time occurrence or occur over time. Expectations are that contributors will submit patches that are small at first and will only grow larger once the contributor has built confidence in the quality of their patches.
+
+*How to become one*: Submit a pull request to a VoxPupuli project
+
+
+Contributors
+------------
+
+
+Contributors are collaborators who have shown wide dedication to the VoxPupuli project in general or deep dedication to one project in particular, and the ability to work well with collaborators and users. The contributors have responsibilities beyond the collaborators. In particular, contributors formally decide on whether a pull request is merged or a release is cut. A contributor will use lazy consensus to decide on whether to merge a pull request from a collaborator. If the discussion is no longer moving towards a consensus, the PMC must vote via lazy consensus on whether the patch should be applied.
+
```

Comment:
```irc
<igalic> nibalizer: so, i meant: usually we defer to experts to provide… expertise on whether something should be merged or not, rather than defering to a PMC to decide such a thing
<igalic> if a decision cannot be made by mere mortals…
<nibalizer> ah like folk who actually develop and run corosync you mean
<igalic> yup
<igalic> even if they are not necessarily within our contributors
<igalic> (although, there are lots of ppl who run it)
<igalic> but, yes
<igalic> (and then, those folks are suddenly experts, and are given merge access ;)
<nibalizer> hrm okay
<nibalizer> yea i agree
<nibalizer> i think when we get loggjamed on our own bullshit we need the pmc
<bastelfreak> igalic: I share your opinion here
<nibalizer> but when there is a clear expert in the room we can just do what they think
<nibalizer> kindof the exact definition of bike shed vs nuclear reactor
<igalic> perfect
<igalic> now we need a smart person to put this in words
```